.eslintrc.js

```diff
@@ -31,7 +31,16 @@ module.exports = {
 		"no-negated-condition": "off",
 		"require-jsdoc": "off",
 		"no-mixed-requires": "off",
-		"max-len": ["warn", 120],
+		"max-len": [
+			"error",
+			{
+				"code": 120,
+				"comments": 200,
```

Comment:
Why such a big difference between code and comments?

Reply:
Removed